src/main/java/org/icgc/argo/program_service/repositories/BaseRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.icgc.argo.program_service.repositories;
+
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.NoRepositoryBean;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+import java.util.Collection;
+import java.util.Set;
+
+@NoRepositoryBean
+public interface BaseRepository<T, ID> extends PagingAndSortingRepository<T, ID>, JpaSpecificationExecutor {
```

Comment:
PagingAndSortingRepository could be replaced with JpaRepository, it's more specific. And btw, this class looks stupid

Reply:
sounds good!